scripts/generate/public_git.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/sh
+#
+# Generates support file from a public git repo
+
+set -e
+
+branch="master"
+directive_map=""
+match_func=""
+url=""
+path=""
+
+help() {
+    cat << EOF
+Generate support file from a git repo.
+
+usage: $(basename "$0") [-d|--directive-map] [-b|--branch] [--url] [-m|--match-func] [-o|--override] [-p|--path] [h|--help]
+    -h  | --help               Display this message
+    -d  | --directive-map      Name of the generated directive map(required)
+    -b  | --branch             Branch to checkout, defaults to "$branch"(optional)
+    -m  | --match-func         Name of the generated MatchFunc(required)
+    -p  | --path               The path of the output file(required)
+    --url                      Url used for git clone(required)
+EOF
+}
+
+while [ ! $# -eq 0 ]; do
+    case "$1" in
+        --help | -h)
+            help
+            exit 0
+            ;;
+        --directive_map | -d)
```

Comment:
the help function says `--directive-map` instead of `--directive_map`, which is right? same for match_func

Reply:
Oh correct. Sorry for such minor error